packages/simlog/simlog.0.0.4/opam

```diff
@@ -0,0 +1,43 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "The mediocre OCaml logging library"
+description: "The mediocre OCaml logging library"
```

Comment:
```suggestion
description: "A really crude and simple OCaml Log Library."
```
I copied this from the repo, but a more elaborate description of the library would be better. Otherwise I would drop the description altogether since it is not really informative.